templates/Web/Projects/VueDefault/README_postaction.md

```diff
@@ -0,0 +1,56 @@
+﻿## Getting Started
+
+In the root directory of the project...
+
+1. Install node modules `yarn install` or `npm install`.
+2. Start development server `yarn start` or `npm start`.
+
+## Next Steps
+//{[{
+### Adding a New Page
+
+1. Create a file in `/src/views` with your Vue Template.
+2. Add a route for your page to `/src/router/index.js`.
+3. Add a button to the navigation bar in `/src/components/NavBar.vue`.
```

Comment:
Good catch forgot to update this. Well the reason i used TheNavBar is since vue style guidelines recommend when theres only one instance possible to add the prefix "The" to the name.

Reply:
Will update this right now before my class starts.

Reply:
Made the edit.